src/development/LocalTest/appsettings.json

```diff
@@ -36,8 +36,8 @@
   "AllowedHosts": "*",
   "LocalPlatformSettings": {
     "LocalTestingStorageBasePath": "c:/AltinnPlatformLocal/",
-    "AppRepsitoryBasePath": "C:/Users/sekeberg/git/steffens-2020-v2/App/",
```

Comment:
No need to check in the changes to this file, but no harm done by including it either..

Reply:
Yeah, accidental. Letting it go.